buildscripts/make_dependencies.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Build protoc & netty
+set -ev
+
+# Make protoc
+pushd .
+cd /tmp
+git clone https://github.com/google/protobuf.git
```

Comment:
do we need to bother with make check btw?, we're only going to be using verified releases